### DISCUSSION TITLE
How to install dutch version of Microsoft 365 Apps for enterprise with winget

### DISCUSSION THREAD

**Discussion by Macleykun** [microsoft/winget-cli#1876] [General] [answered]
I want to install only the dutch version of Microsoft 365 Apps for enterprise but i can't figure it out using --local.

Can someone point me out what the right command is?

so far i tried:
`winget install 'Microsoft 365 Apps for enterprise' --local 'nl-NL'`

**Comment by jedieaston** [comment, marked as answer]
You can't do it with the existing manifest (which was created as a hackathon project at MS), but if you create a config file yourself (https://config.office.com/deploymentsettings), you can use it with the winget package:

`winget install -s winget Microsoft.Office --custom "/configure <path to configuration file>"`

An example config file for you would be:
```
<Configuration ID="bff6cc33-b2e6-471d-949d-faf53f4d08c1">
  <Add OfficeClientEdition="64" Channel="Current">
    <Product ID="O365ProPlusRetail">
      <Language ID="nl-nl" />
      <ExcludeApp ID="Groove" />
      <ExcludeApp ID="Lync" />
    </Product>
  </Add>
  <Updates Enabled="TRUE" />
  <RemoveMSI />
</Configuration>
```
Save that as a .xml file, and put the path to it in the command above.

**Reply by Macleykun** [reply]
Thx! Given the way things are, I’ll likely just remove the en-us after installing both.

**Comment by GoByeBye** [comment]
I'd like to add to this as I was running through the same problem myself and found this discussion through google.

@jedieaston's answer is mostly right. It was probably correct at the time of his writing. 
Currently the parameter `--custom` does not exist. Instead, use `--override` to pass arguments to the installer

For any future lost souls, at the time of writing this would be the correct command

`winget install "Microsoft.Office" --override "/configure C:/path/to/config/file.xml"`
